src/main/webapp/dataUse/DataUseRestriction.js

```diff
@@ -0,0 +1,179 @@
+import React, { Component } from 'react';
+import { h, h1, div, a, hh } from 'react-hyperscript-helpers';
+import { DataUse } from "../util/ajax";
+import { spinnerService } from "../util/spinner-service";
+import { Spinner } from "../components/Spinner";
+import { TableComponent } from "../components/TableComponent";
+import { RESTRICTION_SORT_NAME_INDEX, styles } from "../util/ReportConstants";
+import { TABLE_ACTIONS } from "../util/TableUtil";
+import { handleRedirectToConsentGroup, isEmpty } from "../util/Utils";
+import { format } from 'date-fns';
+
+const SIZE_PER_PAGE_LIST = [
+  { text: '50', value: 50 },
+  { text: '100', value: 100 },
+  { text: '500', value: 500 }];
+
+const columns = [
+  {
+    dataField: 'consentGroupKey',
+    text: 'Consent Group',
+    sort: true,
+    formatter: (cell, row, rowIndex, colIndex) =>
+    div({},[
+      a({ href: handleRedirectToConsentGroup(component.serverURL, row.consentGroupKey) },[row.consentGroupKey])
```

Comment:
I'm a little confused about the direct linking here. Does this reload the page at the destination link or is it intercepted by the router? If I understand our use of the router correctly, these should instead be `Link` objects that point to the correct destination.